<ui version="4.0">
 <class>TauDevPanel</class>
 <widget class="QMainWindow" name="TauDevPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="attrDW">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Attributes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout">
     <item>
      <widget class="TaurusAttrForm" name="tauAttrForm">
       <property name="useParentModel">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="commandsDW">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Commands</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="TaurusCommandsForm" name="tauCommandsForm">
       <property name="useParentModel">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="trendDW">
   <property name="windowTitle">
    <string>Trends</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout">
     <item>
      <widget class="TaurusTrend" name="tauTrend"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
  </customwidget>
  <customwidget>
   <class>TaurusPlot</class>
   <extends>QwtPlot</extends>
   <header>taurus.qt.qtgui.plot</header>
  </customwidget>
  <customwidget>
   <class>TaurusTrend</class>
   <extends>TauPlot</extends>
   <header>taurus.qt.qtgui.plot</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>TaurusCommandsForm</class>
   <extends>TauWidget</extends>
   <header>taurus.qt.qtgui.panel</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaurusAttrForm</class>
   <extends>TauWidget</extends>
   <header>taurus.qt.qtgui.panel</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

